modules/veteran/spec/models/veteran/service/representative_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-describe Veteran::Service::Representative do
+describe Veteran::Service::Representative, vcr: true do
```

Comment:
woah never seen this, only the explicit vcr block, this is cool

Reply:
it let's us capture any future tests that we need to capture and if a test doesn't call out then it just skips over it etc.